esp/templates/users/userchecklist.html

```diff
@@ -14,6 +14,21 @@
 <br />
 <br />
 
+<script type="text/javascript">
+    function checkAll(checkId){
+        var inputs = document.getElementsByTagName("input");
+        for (var i = 0; i < inputs.length; i++) { 
+            if (inputs[i].type == "checkbox" && inputs[i].id == checkId) { 
+                if(inputs[i].checked == true) {
```

Comment:
Here, you can just do `if (inputs[i].checked) {` (and `} else {` below)!  (Aso, while you're at it, if you could fix the whitespace -- our style is spaces after the `if`, no spaces before the `;`.)